src/app/create-event/page.tsx

```diff
@@ -0,0 +1,159 @@
+'use client';
+
+import { NextPage } from 'next';
+import nookies from 'nookies';
+import { format } from 'date-fns';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { EventInput } from './EventInput';
+import { EventTextarea } from './EventTextarea';
+import { postEvent } from '../../api/events';
+import { eventSchema } from './schema';
+
+const CreateEventPage: NextPage = () => {
+  const {
+    register,
+    watch,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(eventSchema),
+  });
+
+  const { token } = nookies.get();
+
+  const onSubmit = async (data: any) => {
+    const dateTimeIso = format(new Date(data.date), "yyyy-MM-dd'T'HH:mm:ss'Z'");
+
+    const eventDataFormatted = {
+      ...data,
+      date: dateTimeIso,
+    };
+
+    if (token) {
+      postEvent(eventDataFormatted, token);
```

Comment:
The event certainly must be added into some `ky` context. Try to google something around it. The same you would use an interceptor in axios, there has to be a similar solution in `ky`